atomic_reactor/core.py

```diff
@@ -214,8 +215,47 @@ def build_image_privileged_container(self, build_image, json_args_path):
         return container_id
 
 
+def retry(function, *args, **kwargs):
+    retry_times = kwargs.pop('retry', None)
+    retry_delay = DOCKER_BACKOFF_FACTOR
+    retry_client_statuses = DOCKER_CLIENT_STATUS_RETRY
+
+    for counter in range(int(retry_times)+1):
```

Comment:
If `retry_times` ends up being None due to `pop` call earlier, this will throw an exception.

It's a little messy to convert this parameter to `int` multiple times in this function.
Would it make more sense to default to `0` instead? This for-loop would still execute once as expected.

Reply:
ok will make it 0 by default and use 'int' just once